tests/unit/test_finder.py

```diff
@@ -469,8 +468,9 @@ class TestCandidateEvaluator(object):
     def setup(self):
         self.search_name = 'pytest'
         self.canonical_name = 'pytest'
-        valid_tags = pip._internal.pep425tags.get_supported()
-        self.evaluator = CandidateEvaluator(valid_tags=valid_tags)
+        target_python = TargetPython()
+        target_python._valid_tags = pip._internal.pep425tags.get_supported()
```

Comment:
This line seems unnecessary since it is the default ?

Reply:
Good point! I'll update..

Reply:
Actually, it looks like creating and passing in the `target_python` can also be removed.